test/python/test_reactor.py

```diff
@@ -220,10 +221,9 @@ def integrate(atol, rtol):
 
         n_baseline = integrate(1e-10, 1e-20)
         n_rtol = integrate(5e-7, 1e-20)
-        n_atol = integrate(1e-10, 1e-6)
-
-        self.assertTrue(n_baseline > n_rtol)
-        self.assertTrue(n_baseline > n_atol)
+        n_atol = integrate(1e-10, 1e-5)
+        self.assertTrue(n_baseline > n_rtol, f"{n_baseline} is less than {n_rtol}")
+        self.assertTrue(n_baseline > n_atol, f"{n_baseline} is less than {n_atol}")
```

Comment:
This is even simpler if you make use of the fact that we're using `pytest` now (which will use introspection magic to provide nice test failure messages):
```suggestion
        assert n_baseline > n_rtol
        assert n_baseline > n_atol
```